.devcontainer/Dockerfile

```diff
@@ -3,6 +3,5 @@ ARG VARIANT=3.13-bookworm
 FROM mcr.microsoft.com/vscode/devcontainers/python:${VARIANT}
 COPY requirements.txt /tmp/pip-tmp/
 RUN python3 -m pip install --upgrade pip \
-  && python3 -m pip install --no-cache-dir install -r /tmp/pip-tmp/requirements.txt \
-  && pipx install pre-commit ruff \
-  && pre-commit install
+  && python3 -m pip install --no-cache-dir -r /tmp/pip-tmp/requirements.txt \
+  && pipx install pre-commit ruff
```

Comment:
`pre-commit may already exist in the dev container.

In any case, `pre-commit install` is still needed.

Reply:
Oh, I see.  This is done below.